app/types/screen.ts

```diff
@@ -1 +1 @@
-export type ScreenType = (navigation: any) => void
+export type ScreenType = (navigation: any) => any
```

Comment:
a type `(any) => any` is not really helpful. it makes the syntax more boiler-platy without adding any type safety. Typescript would tread no type information as any IIRC